ts/src/sdl/sdl-schema.yaml

```
@@ -404,6 +410,7 @@ properties:
       required:
         - image
       type: object
+      additionalProperties: false
```

YAMLlint (1.38.0)

[error] duplication of key "additionalProperties" in mapping

(key-duplicates)